prisma/schema.prisma

```diff
@@ -22,10 +22,11 @@ model ConnectInstallation {
 }
 
 model AssociatedFigmaDesign {
-  id                    BigInt              @id @default(autoincrement())
-  fileKey               String              @map("file_key")
-  nodeId                String              @map("node_id") // Prisma does not support nullable columns with the composite indexes: https://github.com/prisma/prisma/issues/3197
-  associatedWithAri     String              @map("associated-with-ari")
+  id                BigInt  @id @default(autoincrement())
+  fileKey           String  @map("file_key")
+  nodeId            String  @map("node_id") // Prisma does not support nullable columns with the composite indexes: https://github.com/prisma/prisma/issues/3197
+  associatedWithAri String  @map("associated-with-ari")
+  inputUrl          String? @map("input_url")
```

Comment:
**note:** Adding a new column.

If you see more changes in this file, please, click "Hide whitespace" on the toolbar. It seems Prettier reformatted this file a bit.